### PR TITLE
Implements Popular Tables and Table Detail APIs

### PR DESCRIPTION
**Pull request by verdan**
Includes the basic implementation of Popular Tables and Table Detail APIs.
**Popular Tables**:
For now it simply returns all the available (Amundsen Compatible) tables, as we'd refactor and improve that over the time, once we'll generate all the fields to atlas needed for the Amundsen project. 

**Table Detail**:
Simply returns the basic information of a table at the moment and will get mature over the time.

Note: Apache Atlas python client is not very mature at the moment, so I also opened a couple of PR in that project (also added that as comments) and hence I need to rely heavily on the Atlas REST API structure and the way atlas client is providing the data. Once my PRs would be merged, the number of API calls/queries would be much lower. 

Also adds the documentation on how to make atlas tables appear in Amundsen, along with code snippets needed to generate the Super Type and assign that super type to required entity definitions. 

Coverage:
Master: 67%
This branch: 71% (covers all the lines in atlas proxy, excluding `pass` statements)

### CODE REVIEW

**Comment by verdan** [comment]
@feng-tao @jinhyukchang ready for review

**Comment by feng-tao** [comment]
nice @verdan , will take a look. BTW, are you able to get the frontend table detail page shown with the atlas proxy?

**Comment by verdan** [comment]
Screen Shot 2019-03-26 at 8 31 40 PM
Screen Shot 2019-03-26 at 8 31 47 PM

@feng-tao yes, I was able to populate the Popular Tables and Table Detail pages. Please ignore the test data in the screenshots as I didn't populate all the fields yet in atlas, but atlas_proxy points to the correct attributes.

**Comment by feng-tao** [comment]
great news @verdan !

**Comment by feng-tao** [comment]
@verdan I am in the strata conference this week, so my review will be slow. cc @jinhyukchang  and see if he has cycle. Otherwise I will try to get to it end of the week.

File: metadata_service/proxy/atlas_proxy.py [hunk 1]
```diff
@@ -64,7 +144,39 @@ def get_column_description(self, *,
 
     def get_popular_tables(self, *,
                            num_entries: int = 10) -> List[PopularTable]:
-        return []
+        """
+        FixMe: For now it simply returns ALL the tables available,
```
Comment (feng-tao) [inline metadata_service/proxy/atlas_proxy.py]: sounds good. I was wondering where do you define the logic for popular table.

**Pull request merged**